compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -919,7 +919,26 @@ OMR::Power::TreeEvaluator::mTrueCountEvaluator(TR::Node *node, TR::CodeGenerator
 TR::Register*
 OMR::Power::TreeEvaluator::mFirstTrueEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    {
-   return TR::TreeEvaluator::unImpOpEvaluator(node, cg);
+   TR::Node *firstChild = node->getFirstChild();
+
+   TR_ASSERT_FATAL_WITH_NODE(node, firstChild->getDataType().getVectorLength() == TR::VectorLength128,
+                             "Only 128-bit vectors are supported %s", node->getDataType().toString());
+
+   TR_ASSERT_FATAL_WITH_NODE(node, firstChild->getDataType().getVectorNumLanes() == 16,
+                             "Unsupported vector type %s for mToLongBits\n", firstChild->getDataType().toString());
+
+
+   TR::Register *srcReg = cg->evaluate(firstChild);
+   TR::Register *tmpReg = cg->allocateRegister(TR_VRF);
+   TR::Register *resReg = cg->allocateRegister();
+
+   generateTrg1Src1Instruction(cg, OMR::InstOpCode::vclzlsbb, node, resReg, srcReg);
```

Comment:
have a general concern maybe unrelated to this in general:  `vclzlsbb` is checking `bit[7]`, while `vextract*m` are checking `bit[0]`.  they appear to be incompatible ... might be they have different types of input (lane-mask vs.  true-false array???)

Reply:
Yes, that's true. But both fit our purposes for `mFirstTrue `and` mLongToBits` correspondingly.

Reply:
Actually, lane mask in Vector API is a true-false array essentially.